source/hassio/installation.markdown

```diff
@@ -12,7 +12,7 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 3 Model B and B+ 32-bit][pi3-32] (32-bit is required for GPIO support)
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
      - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (32-bit is required for GPIO support)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB, 4 GB and 8 GB model) 64-bit][pi4-64] (64-bit is required for 8 GB model)
+     - [Raspberry Pi 4 Model B (1 GB, 2 GB, 4 GB and 8 GB model) 64-bit][pi4-64] (64-bit is required for 8 GB model; 64-bit use more memory than 32-bit version)
```

Comment:
How so?

Reply:
It is almost always true https://bit.ly/2KUfLtF.  In case of HA, it is checked.

Reply:
Well, I don't think we should mention it here. Furthermore, I would like to see how much of a difference it makes and if it is even worth mentioning.

Reply:
Can I help somehow with this?
I was 40% of 2GB RAM RPi4 on Raspbian. Now on this same machine HA OS 64 bit I do have 80%. I know this is not this same OS but 32bit HA OS is not able to be running in my case (USB SSD).

Reply:
Raspbian, using installation method...?

Reply:
Supervised

Reply:
But this documentation is not about supervised, so your numbers might not be a fair comparison.

Reply:
@frenck, I just did want to report a problem I faced to help other community members do not waste their time or at least to know consequences. 
You are right that Raspbian HA installation stopped to be officially supported and because 32bit HassOS on SSD is not working at all I am unable to help anymore to find more "fair comparision". Hope there is somebody who can compare.